src/IECoreDelight/Renderer.cpp

```diff
@@ -1188,6 +1212,28 @@ class DelightRenderer final : public IECoreScenePreview::Renderer
 					m_oversampling = 9;
 				}
 			}
+			else if( name == g_importancesamplefilterOptionName )
+			{
+				if( value )
+				{
+					if( const IntData *d = reportedCast<const IntData>( value, "option", name ) )
+					{
+						if( m_importancesamplefilter != d->readable() )
+						{
+							stop();
```

Comment:
There's a comment in `output()` about it being necessary to call `stop()` before modifying the output chain because 3delight will crash otherwise, but I wrote that a _long_ time ago. I'd be interested to see if it was still necessary now.

Reply:
Removing stop() from output() does not crash 3Delight any more when changing output parameters, but any changes to the outputs during interactive rendering fully re-initializes the render, so technically there is no big gain from removing stop() from output() unfortunately.

Reply:
Makes sense - thanks for taking the time to check.

Reply:
Removed the stop() function and the comment and committed the changes.